apps/storybook/src/Table.stories.tsx

```diff
@@ -841,7 +841,6 @@ export const LazyLoading: Story<Partial<TableProps>> = (args) => {
 
   return (
     <Table
-      enableVirtualization
```

Comment:
am... I think it does not work with virtualization enabled.
Parent element of `rowRef` there is not the body one, so the check is failing...
Should we add same logic of scrollable element as in VS or handle virtualized table explicitly (by going further than parent element)

Reply:
does intersection observer even make sense with virtualization? it's supposed to be for elements that are outside the viewport and in the DOM, but virtualization does not keep those elements in the DOM

Reply:
well, we do have at least 10 elements by default (can change that) which are outside of dom.. SO that gives a little bit of room, IMO..
Its not that complex to adjust parent if virtualization is enabled.

Reply:
added handling if virtualization is enabled